src/easystar.js

```diff
@@ -421,7 +435,11 @@ EasyStar.js = function() {
     };
 
     // Helpers
-    var isTileWalkable = function(collisionGrid, acceptableTiles, x, y) {
+    var isTileWalkable = function(collisionGrid, acceptableTiles, x, y, sourceNode) {
+        if (directionalConditions[x + "_" + y]) {
+            var direction = calculateDirection(sourceNode.x - x, sourceNode.y - y)
+            if (!includes(directionalConditions[x + "_" + y], direction)) return false
```

Comment:
Oh really? I thought `{Array|String}` meant Array of strings 😅  I was confused by https://github.com/prettymuchbryce/easystarjs/pull/39/files#diff-6c33413ee9916790ad0990a90ac1f375R39, as this Array seems to be an Array of numbers. I'll change it :)